### PR TITLE
Initial import

### PR DESCRIPTION
**Pull request by talebipour**
First version of Kafka parquet writer pushed into github.

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          .build();
+
+
+KafkaProtoParquetWriter<SampleMessage> writer =
+        new Builder<>("TestParquetWriter", "topic", consumerConfig, "/tmp/parquetFiles",
+                      SampleMessage.class, SampleMessage.PARSER).threadCount(1)
+                .maxRecordsInFile(1_000_000)
+                .maxFileOpenDuration(15, TimeUnit.MINUTES)
+                .build();
+```
```
Comment (bozorgzadeh) [inline README.MD:31]: Add:
writer.start();

File: src/main/java/ir/sahab/neor/kafka/reader/KafkaProtoParquetWriter.java [hunk 2]
```diff
@@ -302,6 +302,7 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
             ExecutorService executor = Executors.newFixedThreadPool(1);
             try {
                 executor.submit(() -> consumer.poll(0)).get(60, TimeUnit.SECONDS);
+                executor.shutdown();
```
Comment (bozorgzadeh) [inline src/main/java/ir/sahab/neor/kafka/reader/KafkaProtoParquetWriter.java]: Move shutDown() to finally block.

File: README.MD [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+          .put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG,
+                  StringDeserializer.class.getName())
+          .put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, "kafka-sever:9092")
+          .put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "latest")
+          .build();
+
+
+KafkaProtoParquetWriter<SampleMessage> writer =
+        new Builder<>("TestParquetWriter", "topic", consumerConfig, "/tmp/parquetFiles",
+                      SampleMessage.class, SampleMessage.PARSER).threadCount(1)
```
Comment (bozorgzadeh) [inline README.MD]: Move threadCount(1) into the next line (just for better view)

File: README.MD [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Kafka Proto Parquet Writer
+Kafka proto parquet writer reads logs from a Kafka topic and writes them as parquet files. Parquet files can be created on local file system or HDFS. It can read and write logs in multiple threads. As writing to single parquet file can not be done concurrently, each threads writes to a separate file.
```
Comment (bozorgzadeh) [inline README.MD]: logs -> records

This is github! I should indicate the word which I want to change!

Comment (bozorgzadeh) [inline README.MD]: Please change occurrences of 'logs'  to 'records'.

File: README.MD (hunk 1 above)
Comment (talebipour) [reply README.MD:31]: Done

File: src/main/java/ir/sahab/neor/kafka/reader/KafkaProtoParquetWriter.java (hunk 2 above)
Comment (talebipour) [reply src/main/java/ir/sahab/neor/kafka/reader/KafkaProtoParquetWriter.java]: Done

File: README.MD (hunk 3 above)
Comment (talebipour) [reply README.MD]: Done

**Pull request merged**